bl/evaluation/evaluator_test.go

```diff
@@ -15,14 +15,14 @@ type mockCliClient struct {
 	mock.Mock
 }
 
-func (m *mockCliClient) CreateEvaluation(createEvaluationRequest *cliClient.CreateEvaluationRequest) (*cliClient.CreateEvaluationResponse, error) {
-	args := m.Called(createEvaluationRequest)
-	return args.Get(0).(*cliClient.CreateEvaluationResponse), args.Error(1)
+func (m *mockCliClient) RequestPrerunDataForEvaluation(token string) (*cliClient.PrerunDataForEvaluationResponse, error) {
+	args := m.Called(token)
+	return args.Get(0).(*cliClient.PrerunDataForEvaluationResponse), args.Error(1)
 }
 
-func (m *mockCliClient) RequestEvaluation(evaluationRequest *cliClient.EvaluationRequest) (*cliClient.EvaluationResponse, error) {
-	args := m.Called(evaluationRequest)
-	return args.Get(0).(*cliClient.EvaluationResponse), args.Error(1)
+func (m *mockCliClient) SendLocalEvaluationResult(localEvaluationResultRequest *cliClient.LocalEvaluationResultRequest) (*cliClient.SendEvaluationResultsResponse, error) {
```

Comment:
The rules evaluation will always be local, making this information redundant, I suggest renaming to SendEvaluationResult